### PR TITLE
Add utility tools and sample files for demonstration

### PR DESCRIPTION
**Pull request by AuroraHuang22**

### CODE REVIEW

File: nft-book/utils/uploadFiles.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  for (const file of files) {
+    const formData = new FormData();
+    const fileStream = fs.createReadStream(`${basePath}/files/images/${file}`);
+    formData.append("file", fileStream);
+
+    let url = "https://api.nftport.xyz/v0/files";
+    let options = {
+      method: "POST",
+      headers: {
+        Authorization: "",
```
Comment (williamchong) [inline nft-book/utils/uploadFiles.js]: We can use `process.env.XXXXX` to represent a secret that needs to be filled

File: nft-book/utils/convertToCsv.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+function formatIndex(index) {
+  if (index < 10) {
+    return `000${index}`;
+  }
+  if (index === 10 || index < 100) {
+    return `00${index}`;
+  }
+  if (index === 100 || index < 1000) {
```
Comment (williamchong) [inline nft-book/utils/convertToCsv.js]: is the `index === 10` `index === 100` part redundant?

File: nft-book/utils/convertMetas.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+const basePath = process.cwd();
+
+fs.writeFileSync(`${basePath}/output/json/_metadata.json`, "");
+const writer = fs.createWriteStream(`${basePath}/output/json/_metadata.json`, {
+  flags: "a",
+});
+
+writer.write("[");
+const readDir = `${basePath}/output/json`;
+let fileCount = fs.readdirSync(readDir).length - 2;
```
Comment (williamchong) [inline nft-book/utils/convertMetas.js]: Why -2? if you are counting `.DS_Store`, it does not exists on Linux or Windows, so we cannot hard code it

**Review by williamchong — CHANGES_REQUESTED** [review]

File: nft-book/utils/convertToCsv.js (hunk 2 above)
Comment (AuroraHuang22) [reply nft-book/utils/convertToCsv.js]: 🌚

**Review by williamchong — APPROVED** [review]

**Pull request merged**